praw/models/reddit/live.py

```diff
@@ -281,6 +281,58 @@ def close(self):
         url = API_PATH['live_close'].format(id=self.thread.id)
         self.thread._reddit.post(url)
 
+    def update(self, title=None, description=None, nsfw=None, resources=None,
+               **other_settings):
+        """Update settings of the live thread.
+
+        :param title: (Optional) The title of the live thread (default: None).
+        :param description: (Optional) The live thread's description
+            (default: None).
+        :param nsfw: (Optional) Indicate whether this thread is not safe for
+            work (default: None).
+        :param resources: (Optional) Markdown formatted information that is
+            useful for the live thread (default: None).
+
+        Return ``None`` if there is no argument.
+
+        Each setting will maintain its current value if ``None`` is specified.
+
+        Additional keyword arguments can be provided to handle new settings as
+        Reddit introduces them.
+
+        Usage:
+
+        .. code-block:: python
+
+           thread = reddit.live('xyu8kmjvfrww')
+
+           # update `title` and `nsfw`
+           updated_thread = thread.contrib.update(title=new_title, nsfw=True)
+
+        If reddit introduces new settings, you must specify ``None`` for the
```

Comment:
Can we use `Reddit` instead of `reddit`? I know Reddit used to recommend that it should appear all lower-case but that appears to no longer be the case.